readme.md

```diff
@@ -587,12 +567,62 @@ If a signal terminated the subprocess, this property is defined and included in
 
 #### pipedFrom
 
-Type: [`SubprocessResult[]`](#subprocessresult)
+Type: [`Array<SubprocessResult | ExecaError>`](#subprocessresult)
```

Comment:
`SubprocessResult` is a subclass of `ExecaError`, but it's useful to distinguish those to make it clear `pipedFrom` might contain both successful results and error instances.